_posts/2021-06-04-flavourzyme-a-purified-enzyme-mixture-from-a-oryzae.md

```diff
@@ -82,9 +82,39 @@ I won't go into too much detail here, but the final mixture can either be dried
 
 ### The Final Product
 
-If everything's gone well so far we should have a powerful enzyme mixture rated for 500 LAPU/g (one LAPU is the amount of enzyme which hydrolyzes 1 µmol of L-leucine-pnitroanilide per minute - this is one of many enzyme assays for proteolytic enzymes) and between 6% w/w of pure enzyme protein, depending on formulation [6]. By comparison, unrefined shoyu koji contains around 0.2% w/w protease [7].
+If everything's gone well so far we should have a powerful enzyme mixture rated for 500 LAPU/g (one LAPU is the amount of enzyme which hydrolyzes 1 µmol of L-leucine-pnitroanilide per minute - this is one of many enzyme assays for proteolytic enzymes) and 6% w/w of pure enzyme protein, depending on formulation [6]. By comparison, unrefined shoyu koji contains around 0.2% w/w protease [7].
 
-Novozymes A/S won't sell to non-commercial or non-institutional end-users. So naturally, I'd have to import an imitation product from Alibaba. The other question is, of course, can I brew this at home?
+The specific mixture of enzymes identified by researchers is as follows [1]:
+
+- leucine aminopeptidase A
+- leucine aminopeptidase 2
+- dipeptidyl peptidase 4
+- dipeptidyl peptidase 5
+- neutral protease 1
+- neutral protease 2
+- alkaline protease 1
+- α-amylase A type 3
+
+Interestingly, α-amylase ends up present as a by-product - I suppose it's difficult to get *A. oryzae* not to produce any amylases. 
+
+### Applications 
+ 
+Novozymes A/S won't sell to non-commercial or non-institutional end-users, so I had to import an imitation product from Alibaba. Here's what it looks like:
+
+[flavourzyme_photo]
+*A Flavourzyme-like product from Alibaba*
+
+I plan to use this to make ultra-fast garums - which has been attempted by the guys at Noma, but using [pork pancreatic enzymes](https://www.instagram.com/p/Bsd-8kVhQdz/?hl=en). Also, credit goes to Preston at [Culinary Crush](https://www.culinarycrush.biz/all/i-bought-industrial-protease-enzymes) for inspiring this post. A few more applications I will test:
+
+- Accelerated miso / shoyu
+- High-fat miso (flavourzyme contains no lipases) 
+- Accelerated cured tofu 
+- Home-made Maggi sauce (Red-cap maggi sauce sold in Canada is essentaily wheat gluten + enzymes and additives) 
+- Hydrolyzing plant-based milks 
+
+As a quick side note, Japanese brewers today don't add pure enzyme mixtures to misos or shoyus - at least not for more expensive types. The reason is unclear, but it has been tested in the past [8]. 
+
+The other question is, of course, can I brew this at home?
 
```

Comment:
Beefed up the application section a bit. I'm going to give credit to Noma and another blog here if that's OK